frontend/Container.js

```diff
@@ -37,14 +43,42 @@ class Container extends React.Component {
     },
     extraTabs: {[key: string]: () => React$Element},
   };
+  state: State;
+
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      isVertical: (window.innerWidth < 500),
+    };
+  }
+
+  componentDidMount() {
+    window.addEventListener('resize', (e) => this.handleResize(e), false);
```

Comment:
Yeah, that was why I asked about adding a resize :) wasn't sure if you'd want to throttle or not. 100 feels a little slow, 50 seems comparable to the Devtool's element tab.